docs/repositories-configure/using-submodules.md

```diff
@@ -4,6 +4,9 @@
 
 By default, Codacy does normal Git clones that **don't include submodules** to ensure that we only clone necessary repositories. If your organization needs to use submodules, you can request Codacy to enable this feature for you.
 
+!!! note
+    On GitHub organizations, your repository and the repositories that you add as submodules must belong to the same GitHub organization.
```

Comment:
@andrzej-janczak I added the new limitation for GH when using submodules. Let me know if this sentence is clear. 🙏

Reply:
yes, this is true  👍